views/index.html

```diff
@@ -36,8 +36,9 @@
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-two-thirds-from-desktop">
                 <h1 class="govuk-heading-l">Sorry, the service is unavailable</h1>
-                <p class="govuk-body">We are working on making this service available as soon as possible.<p/>
-                <p class="govuk-body">If you would like to transfer digital records in the meantime, please contact <a class="govuk-link" target="_blank" rel="noreferrer noopener" href="mailto:tdr@nationalarchives.gov.uk">tdr@nationalarchives.gov.uk</a>.</p>
+                <p class="govuk-body">The National Archives Transfer Digital Records service is unavailable from <time datetime="2024-12-20">20 December 2024</time> until <time datetime="2025-01-02">2 January 2025</time>.</p>
```

Comment:
Have these dates been confirmed by the business?

Reply:
Claire's saying from 23 Dec to 1st Jan inclusive (with time to deploy/undeploy tbc but "ideal" would be from last thing Fri 20th to, I guess, as early as possible on Jan 2nd ). So to me the sentence we have here makes sense.  Anyway, this is just to test the deploy action to switch on/off works ok so I will take a screen shot and clear the text with her, updating if needed + make a new ticket to actually run it on those dates/times .